### ISSUE TITLE
Next with three.js example not working.

### ISSUE THREAD

**Issue by trevor-ofarrell** [vercel/next.js#20722]
# Bug report
https://github.com/vercel/next.js/tree/canary/examples/with-three-js

## Describe the bug
code breaks when ran. error is "Unhandled Runtime Error
TypeError: uniforms.inverseProjectionMatrix.value.copy(...).invert is not a function"

## To Reproduce

1) yarn create next-app --example with-three-js with-three-js-app

2) yarn && yarn dev

3) click either example(birds and boxes).

## Expected behavior

I see animations

**Comment by ciruz** [comment]
Just as a information for someone trying to fix this.

Everything is a bit outdated in this example, as soon as you upgrade drei ( https://github.com/pmndrs/drei ) to a newer version, StandardEffects got removed ( https://github.com/pmndrs/drei/commit/0642c04204b54520272ad3e07c53ee3629703345 ) and postprocessing as dependencie too.

Prob. relevant: https://discourse.threejs.org/t/can-not-load-gltfloader-in-nextjs-application/12317

**Comment by unphased** [comment]
I got the same `Unhandled Runtime Error TypeError: uniforms.inverseProjectionMatrix.value.copy(...).invert is not a function` and it took a long time to work out the problem, I got sent on a wild goose chase based on observations I made on behavior seen with toggling version numbers in codesandbox. 

This comes down to invert being absent from three.js v122 and earlier. Went away for me after upgrading to 123. In the exceedingly unlikely chance that someone else has the same exact problem (based on the lack of google matches for this error), there's your solution.
